api/service/explorer/service.go

```diff
@@ -174,6 +176,23 @@ func (s *Service) GetAddresses(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// GetHeight serves end-point /addresses, returns size of addresses from address with prefix.
+func (s *Service) GetHeight(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+
+	bc := s.backend.Blockchain()
+	out := map[uint32]uint64{
```

Comment:
Suggest to use slice instead of map. Map is unordered while list is ordered. We may prefer the result sort in shard ID.